addons/web/static/src/js/fields/basic_fields.js

```diff
@@ -2022,7 +2022,8 @@ var JournalDashboardGraph = AbstractField.extend({
                         .y(function (d) { return d.value; })
                         .showValues(false)
                         .showYAxis(false)
-                        .margin({'left': 0, 'right': 0, 'top': 0, 'bottom': 40});
+                        .color(['#875A7B', '#526774', '#FA8072'])
```

Comment:
Are those colors equal to some LESS variables ? We might want to find a way to "load" those LESS colors in JS or at least, comment here with the LESS variable names